library/src/CMakeLists.txt

```diff
@@ -205,11 +205,16 @@ rocm_export_targets(
 
 rocm_install_symlink_subdir( rocblas )
 
+#Control for enabling or disabling header testing
+option (RUN_HEADER_TESTING ON)
```

Comment:
I believe `option` should have the documentation first and then the initial value:

https://cmake.org/cmake/help/latest/command/option.html

If the documentation is missing, then this is treated as `Off`.